### DISCUSSION TITLE
Further development of v4

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#630] [Q&A] [answered]
Does anyone test V5 in real operation and also on longer tours? For me, V5 hangs at least twice on every tour. App no longer responds. Especially if you deviate from the planned route and the route is recalculated. In addition, curves without an angle of inclination (3D) no longer appear as curves, but often as straight lines, especially at a long distance to the next junction. I am not a developer, but I have read that 16KB pages are optional and Android 15 still supports 4 KB. Can we have another meaningful discussion about the wrong horse?

![Screenshot_20240907-164405_Firefox](https://github.com/user-attachments/assets/bc04b605-286d-4c8e-a9d0-1b1f80dbeee1)

**Comment by devemux86** [comment, marked as answer]
Development of v4 is over.

Development is now on v5.

**Comment by devemux86** [comment]
There is no option for apps. Apps are forced to rebuild and use new rules or their updates are rejected.

There is no need to discuss technical details. It's Google's rules.

**Comment by devemux86** [comment]
@DolomitesTom 

For reports please create a new topic and provide your app profile, the route
and detailed information about what happened so we can understand and reproduce it.

**Reply by DolomitesTom** [reply]
I can't remember exactly where that was, I only ran V5 in parallel for about 2000km. After the last hitch, I installed V4 again on the second device. At the risk of making myself unpopular, my motivation to continue using V5 is limited.

**Reply by devemux86** [reply]
@DolomitesTom There is no problem! :slightly_smiling_face:

**Reply by Leeb64** [reply]
I often do small tours ~ 300km and did a slightly larger tour (1700km) with V5.
There were no problems such as crashes.
The "jumping" of the zoom was annoying every now and then!
I think this will get even better!?!
3D would be nice, but not absolutely necessary.

**Comment by devemux86** [comment]
Thank you all for your reports.
More improvements are planned according to your feedback.

Let's close here, as we already have a thread about v5:
- https://github.com/devemux86/cruiser/discussions/559
